FFCPayBatchGenerator/Factory/BPSFactory.cs

```diff
@@ -30,12 +27,11 @@ public override string GetContent()
 
         for (int i = 0; i < batchSize; i++)
         {
-            sb.AppendLine(string.Format("H^SITI{0}^{1}^C{2}^{3}^{4}^{5}^{6}^GBP",
+            sb.AppendLine(string.Format("H^SITI{0}^{1}^C{2}^{3}^1^{4}^{5}^GBP",
                 invoiceNumber.ToString("D7"),
                 requestNumber.ToString("D3"),
                 invoiceNumber.ToString("D7"),
                 frn,
-                requestNumber == 1 ? 1 : 2,
```

Comment:
This was actually a bug.  This value represents `Pillar` which is always `1` for BPS.